packages/api/app/Models/Technology.js

```diff
@@ -116,11 +146,42 @@ class Technology extends Model {
 		return this.belongsTo('App/Models/Upload', 'thumbnail_id');
 	}
 
+	reviewers() {
+		return this.belongsToMany('App/Models/Reviewer');
+	}
+
+	revisions() {
+		return this.hasMany('App/Models/Revision');
+	}
+
 	getOwner() {
 		return this.users()
 			.wherePivot('role', 'OWNER')
 			.first();
 	}
+
+	async checkResponsible(user, role = null) {
+		let responsible = this.users()
+			.select('id')
+			.where({ user_id: user.id });
+		if (role) {
+			responsible = responsible.wherePivot('role', role);
+		}
+		responsible = await responsible.first();
+		return !!responsible;
+	}
+
+	async getTRL() {
+		const stageTaxonomy = await Taxonomy.getTaxonomy('STAGE');
+		const stage = await this.terms()
+			.where({ taxonomy_id: stageTaxonomy.id })
+			.first();
+		const trl = stage ? stage.slug.split('-') : [];
+		if (trl && trl.length) {
+			return trl[1];
+		}
+		return 0;
```

Comment:
Esse if aqui sempre será verdadeiro, não? Quando `stage` for falsy ele vai entrar no if e retornar `undefined`, que é a posição 1 do array vazio.